### DISCUSSION TITLE
Chrome does not authenticate password reset tokens

### DISCUSSION THREAD

**Discussion by developkariyer** [pimcore/pimcore#17529] [Q&A / Support]
Using Pimcore 2024.2, Chrome Version 128.0.6613.85 (Official Build) (64-bit)

Password reset links and Invitation links does not work in Chrome. According to logs, everything is normal but at some point, not visible in debug logs, frontend decides that user is not valid and redirects to login page.

Tested in Brave, links work ok.
Tested in vanilla Pimcore 2024.2, links work in Brave, fails in Chrome.

Also, this seems to be problem in my Slack authenticator, which I was too focused on Pimcore, could not think trying different browsers.

BTW, tried resetting Chrome, cleaning everything, etc... And no extensions in Chrome

Edit: It is weirder. If I click from my e-mail page (gmail) it authenticates, redirects to admin page, then redirects to login.
If I copy and paste the link to any browser, it runs as expected. So I think it is related to Pimcore, especially related to referrer.

**Might it be a BUG in PIMCORE?**

**Comment by phosphordev** [comment]
do your pw reset links have `https` prefix?

**Reply by developkariyer** [reply]
Yes. Indeed, links work when directly copy&pasted.
It still works according to logs but after authentication finishes and user is redirected to admin ui home page, admin ui re-redirects to login page.
Pimcore is running native (not docker) on a standalone Ubuntu server with Apache, mysql, redis etc serving only Pimcore.
